src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -905,6 +907,15 @@ public Map<String, Object> handleAboutToSubmit(CallbackRequest callbackRequest)
 
         caseDataMap.put(CASE_INVITES, generateCaseInvitesForParties(caseData));
         caseDataMap.putAll(setSoaOrConfidentialWaFields(caseData, callbackRequest.getEventId()));
+        String isAllApplicantsAreLiP = (String) caseDataMap.get(WA_IS_APPLICANT_REPRESENTED);
+        log.info("isAllApplicantsAreLiP==> {}", isAllApplicantsAreLiP);
+        if (null == isAllApplicantsAreLiP) {
```

Comment:
Check StringUtils.isEmpty

Reply:
It will be a empty string once SOA done for the first time. so for the consequent time, we shouldnt execute this block. one and only if isApplicantReperented is null we should execute this block.